### PR TITLE
Reordered policies and enabled the all Policies in PSAP

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [x] documentation
- [ ] dependencies

## Description

1. Enabled the policies in PSAP
2. Re-arranged the policies to run
3. Policies running dry_run=no: ```["ec2_idle", "ec2_stop", "unattached_volume", "ip_unattached", "zombie_snapshots", "unused_nat_gateway", "s3_inactive", "empty_roles", "zombie_cluster_resource"]```

### CODE REVIEW

File: jenkins/clouds/aws/daily/policies/Jenkinsfile [hunk 1]
```diff
@@ -1,57 +1,58 @@
+account = ['psap']
 pipeline {
+    options {
+        disableConcurrentBuilds()
+    }
     agent {
         docker {
-            label 'cloud-governance-worker'
+            label 'haim-cloud-governance-worker'
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/policies/Jenkinsfile:8]: Why haim ?

**Review by ebattat — COMMENTED** [review]
Need to review it together, a lot of deletion done by you

File: jenkins/clouds/aws/daily/policies/Jenkinsfile (hunk 1 above)
Comment (athiruma) [reply jenkins/clouds/aws/daily/policies/Jenkinsfile]: Oh my bad. Fixed it.

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 2]
```diff
@@ -1,141 +1,132 @@
...
 users_manager_mails = os.environ['USERS_MANAGER_MAILS']
+LOGS = os.environ.get('LOGS', 'logs')
 account_admin = os.environ['ACCOUNT_ADMIN']
-LDAP_HOST_NAME = os.environ['LDAP_HOST_NAME']
 ES_HOST = os.environ['ES_HOST']
 ES_PORT = os.environ['ES_PORT']
 ES_INDEX = os.environ.get('ES_INDEX')
+GOOGLE_APPLICATION_CREDENTIALS = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+SPREADSHEET_ID = os.environ['AWS_IAM_USER_SPREADSHEET_ID']
+GITHUB_TOKEN = os.environ['GITHUB_TOKEN']
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/run_policies.py:70]: Shift to Starting

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 3]
```diff
@@ -1,141 +1,132 @@
...
+            if not filename.startswith('__') and filename.endswith(file_type):
+                if filename.split('.')[0] not in exclude_policies:
+                    if not file_type:
+                        custodian_policies.append(os.path.splitext(filename)[0])
+                    elif file_type and file_type in filename:
+                        custodian_policies.append(os.path.splitext(filename)[0])
+    return custodian_policies
+
+
+exclude_policies = ['cost_explorer', 'optimize_resources_report', 'monthly_report', 'cost_over_usage',
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/run_policies.py:25]: exlude_global_policies

File: jenkins/clouds/aws/daily/policies/Jenkinsfile [hunk 4]
```diff
@@ -7,51 +11,48 @@ pipeline {
...
         REPLY_TO = credentials('cloud-governance-reply-to')
-        LDAP_HOST_NAME = credentials('cloud-governance-ldap-host-name')
-        ES_INDEX = credentials('cloud-governance-es-index')
-
         contact1 = "ebattat@redhat.com"
         contact2 = "athiruma@redhat.com"
+        // Find the all available policies: https://github.com/redhat-performance/cloud-governance/tree/main/cloud_governance/policy
+        // By default, all policies are running in dry_run="yes" mode and the whole list can be found in run_policies.py
+        // POLICIES_IN_ACTION: Policies that run in the dry_run="no" mode
+        POLICIES_IN_ACTION = '["ec2_idle", "ec2_stop", "unattached_volume", "ip_unattached", "zombie_snapshots", "unused_nat_gateway", "s3_inactive", "empty_roles", "zombie_cluster_resource"]'
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/Jenkinsfile:30]: Shift this to first place in env

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 5]
```diff
@@ -1,141 +1,132 @@
...
+                    'skipped_resources', 'cost_explorer_payer_billings', 'cost_billing_reports', 'spot_savings_analysis']
+available_policies = get_policies(exclude_policies=exclude_policies)
+# # available_policies: Run policies in dry_run="yes" mode
+
+
+def run_cmd(cmd: str):
+    """
+    This method run the command
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/run_policies.py:33]: this method runs the shell command

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 6]
```diff
@@ -1,141 +1,132 @@
...
+    :type cmd:
+    :return:
+    :rtype:
+    """
+    os.system(cmd)
+
+
+def get_container_cmd(env_dict: dict):
+    def generate_container_env(item):
+        return f'-e {item[0]}="{item[1]}"'
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/run_policies.py:44]: Move to up or write the lambda function inside the map

File: jenkins/clouds/aws/daily/policies/run_policies.py (hunk 3 above)
Comment (athiruma) [reply jenkins/clouds/aws/daily/policies/run_policies.py:25]: GLOBAL_POLICIE=["s3_inactive", "empty_roles"]

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 7]
```diff
@@ -1,141 +1,132 @@
...
+            container_cmd = ''
+            if policy in ('empty_roles', 's3_inactive') and region == 'us-east-1':
+                container_cmd = get_container_cmd(container_env_dict)
+            else:
+                if policy not in ('empty_roles', 's3_inactive'):
+                    container_cmd = get_container_cmd(container_env_dict)
+            if container_cmd:
+                run_cmd(container_cmd)
+
+
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/run_policies.py:107]: Add comment 
# Running the policies in dry_run=yes

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 8]
```diff
@@ -1,141 +1,132 @@
...
+run_cmd(f"echo Running the cloud_governance policies with dry_run=yes")
+run_cmd(f"echo Polices list: {policies_not_action}")
+run_policies(policies=policies_not_action)
 
+run_cmd('echo "Running the CloudGovernance policies with dry_run=no" ')
```
Comment (athiruma) [inline jenkins/clouds/aws/daily/policies/run_policies.py:112]: Add comment dry_run=no

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**